### ISSUE TITLE
Link front-end with smart contracts

### ISSUE THREAD

**Issue by tikimeat** [strange-labs-uk/ethereum-lottery#6]
Once the smart contracts have been developed, they will need to be integrated with the front-end. Suggested location for the integration code: `/frontend/public/js/lottery.js.`

**Comment by ccdle12** [comment]
Great! all we need is an injection from testrpc or private chain for the web3 instance and then the front end can monitor events emitted from the contract.

Using the MetaMask web3 instance is buggy at the moment in terms of monitoring the logs

**Comment by ccdle12** [comment]
I've updated lottery.js to use the web3 instance from testrpc || truffle develop || a private chain.

We will still need to paste in the address of the deployed contract. At the moment on start the front end retrieves the deployed contract using an address and calls _goal()_ on the deployed contract to retrieve the goal of the fund raising contract.

It then updates the UI with return value. I've kept the functions very small and simple for now, so we can always decide later on styling and organization of the function calls.
